UM/Qt/Bindings/FileProviderModel.py

```diff
@@ -59,9 +60,12 @@ def update(self) -> None:
                     "name": plugin_id,
                     "displayText" : file_provider.menu_item_display_text,
                     "fileProvider": file_provider,
-                    "shortcut": file_provider.shortcut
+                    "shortcut": file_provider.shortcut,
+                    "priority": file_provider.getPriority(),
```

Comment:
Convert the priority into a property of the FileProvider
```suggestion
                    "priority": file_provider.priority,
```